src/Trixi.jl

```diff
@@ -80,6 +80,7 @@ using Preferences: @load_preference, set_preferences!
 
 const _PREFERENCE_SQRT = @load_preference("sqrt", "sqrt_Trixi_NaN")
 const _PREFERENCE_LOG = @load_preference("log", "log_Trixi_NaN")
+const _PREFERENCE_POLYESTER = parse(Bool, @load_preference("polyester", "true"))
```

Comment:
```suggestion
const _PREFERENCE_POLYESTER = @load_preference("polyester", true)
```
Why not store the preference as Bool in the first place?

Reply:
Latent paranoia, but yes this ought to be fine.
I think I was remembering environment variables and the plethora of variants for yes/no/on/off/0/1